src/open_prime_rando/dol_patching/echoes/dol_versions.py

```diff
@@ -85,6 +86,13 @@
             map_icon_jumptable=0x803B3638,
         ),
         double_damage_vfx=0x80017F28,
+        widescreen_render=WidescreenRenderAddresses(
+            culling_original_instruction_address=0x8030256C,
+            culling_custom_frustum_value_offset=-0x3490,  # assume r2 = 0x804223c0
```

Comment:
r2 is already tracked via the `sda2_base` variable, meaning you could have these two offsets be just addresses and do the math in runtime.

Also remove the suffix `_address` from all attributes because everything is an address.